ClientTests/ImageLoadingHandlerTests.swift

```diff
@@ -0,0 +1,34 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+@testable import Client
+
+import XCTest
+import SDWebImage
+
+class ImageLoadingHandlerTests: XCTestCase {
+
+    override func setUp() {
+        super.setUp()
+        let defaults = UserDefaults.standard
+        defaults.removeObject(forKey: SDWebImageCacheKey.hasClearedCacheKey)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    func test_SDWebImageDiskCacheClear() {
+        let expectation = expectation(description: "Wait for SDWebImage disk cache to clear")
+
+        SDImageCache.shared.clearDiskCache { didClear in
+            XCTAssertTrue(didClear)
+            expectation.fulfill()
```

Comment:
Can we add a check for the user defaults value since you are resetting in setUp function?